Formula/jetty-runner.rb

```diff
@@ -17,6 +17,8 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jetty-runner --version", 1)
```

Comment:
Thanks! The additional test lines do not provide much benefit imho and just make the test more fragile.

Adding a .war that then can be used in the test is not that difficult, see for example [apktool.rb](https://github.com/Homebrew/homebrew-core/blob/e8946db1b1fe8a2d11a82d99683ed710b9d2da4a/Formula/apktool.rb#L9-L12). Maybe https://github.com/efsavage/hello-world-war does the trick?